packages/reakit/src/Radio/Radio.ts

```diff
@@ -19,9 +20,9 @@ export type RadioOptions = RoverOptions &
 
 export type RadioHTMLProps = RoverHTMLProps & React.InputHTMLAttributes<any>;
 
-export type RadioProps = RadioOptions & RadioHTMLProps;
+export type RadioProps<T> = RadioOptions<T> & RadioHTMLProps;
 
-export const useRadio = createHook<RadioOptions, RadioHTMLProps>({
+export const useRadio = createHook<RadioOptions<string>, RadioHTMLProps>({
```

Comment:
I think it is, but since we can't do `Hook<RadioOptions<_fill_this_later_>, RadioHTMLProps>` (https://github.com/microsoft/TypeScript/issues/1213) this would require inlining result of Hook.

```ts
const useRadio: {
  <T>(options?: RadioOptions<T>, htmlProps?: RadioHTMLProps): RadioHTMLProps;
  __keys: ReadonlyArray<any>;
  __useOptions: <T>(options: RadioOptions<T>, htmlProps: RadioHTMLProps) => RadioOptions<T> ;
  __propsAreEqual?: <T>(prev: RadioOptions<T> & RadioHTMLProps, next: RadioOptions<T> & RadioHTMLProps) => boolean;
} = createHook<RadioOptions, RadioHTMLProps>(...)
```